src/styles/base.scss

```diff
@@ -1555,3 +1555,592 @@ td, th {
 .ev-fixed {
   position: fixed;
 }
+
+.display-small {
+  display: none;
+}
+
+/* autofill transparent */
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active {
+  transition: background-color 5000s ease-in-out 0s;
+  -webkit-text-fill-color: #FFFFFF !important;
+}
+
+input.dark-autofill:-webkit-autofill,
+input.dark-autofill:-webkit-autofill:hover,
+input.dark-autofill:-webkit-autofill:focus,
+input.dark-autofill:-webkit-autofill:active {
+  transition: background-color 5000s ease-in-out 0s;
+  -webkit-text-fill-color: #4d4d4d !important;
+}
+
+input[type="radio"].selectTeam + label {
+  font-size: 15px;
+  color: $med-black;
+  vertical-align: middle;
+}
+
+[type="radio"]:checked + label:after,
+[type="radio"].with-gap:checked + label:after {
+  background-color: #252833;
+}
+
+[type="radio"]:checked + label:after,
+[type="radio"].with-gap:checked + label:before,
+[type="radio"].with-gap:checked + label:after {
+  border: 2px solid #252833;
+}
+
+body,
+html {
+  font-family: 'Roboto';
+}
+
+a {
+  color: $med-gray;
+}
+
+p.no-top-para {
+  margin-top: 2px;
+}
+
+.rm-row-bot {
+  margin-bottom: 0px;
+}
+
+span.inline {
+  p {
+    display: inline;
+  }
+}
+
+/*clearfix*/
+
+.clearfix {
+  clear: both;
+  content: '';
+}
+
+/*bg color*/
+
+.white-bg {
+  background-color: #fff;
+}
+
+.ev-dark-bg {
+  background-color: $dark-gray;
+}
+
+.ev-super-light-bg {
+  background-color: $grad-super-light;
+}
+
+/*main intro page*/
+
+.ev-sm-container {
+  padding: 10px 20px;
+}
+
+.padding-lr-header {
+  padding: 0px 100px;
+}
+
+a.active {
+  color: $link-hover-dark;
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+.ev-btn-dark:focus {
+  background-color: $dark-gray !important;
+}
+
+nav {
+  ul {
+    li {
+      a {
+        transition: all 0.2s ease-in-out;
+        &:hover {
+          background-color: rgba(0, 0, 0, 0);
+        }
+      }
+    }
+  }
+}
+
+.ev-container {
+  width: 100%;
+  padding: 20px 40px;
+  position: relative;
+  color: #fff;
+}
+
+.w-300 {
+  font-weight: 300;
+}
+
+.w-400 {
+  font-weight: 400;
+}
```

Comment:
@sanketbansal No need to add these fonts weight as they are already present here https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L127. You can use these